src/Common/Dwarf.cpp

```diff
@@ -836,7 +836,7 @@ Dwarf::CompilationUnit Dwarf::findCompilationUnit(uint64_t targetOffset) const
         chunk.remove_prefix(offset);
 
         auto initial_length = read<uint32_t>(chunk);
-        auto is64_bit = (initial_length == static_cast<uint32_t>(-1));
+        auto is64_bit = std::cmp_equal(initial_length, static_cast<uint32_t>(-1));
```

Comment:
Why do we need std::cmp_equal if types are the same? (here and everywhere else in this file)